specification/compute/resource-manager/Microsoft.Compute/CloudserviceRP/stable/2022-04-04/cloudService.json

```diff
@@ -2090,19 +2090,19 @@
       "description": "Describes the properties of the load balancer configuration.",
       "type": "object",
       "properties": {
-        "frontendIPConfigurations": {
+        "frontendIpConfigurations": {
           "description": "Specifies the frontend IP to be used for the load balancer. Only IPv4 frontend IP address is supported. Each load balancer configuration must have exactly one frontend IP configuration.",
           "type": "array",
           "items": {
-            "$ref": "#/definitions/LoadBalancerFrontendIPConfiguration"
+            "$ref": "#/definitions/LoadBalancerFrontendIpConfiguration"
```

Comment:
You actually do not need to change this, this change breaks the references

Reply:
so can i close this PR?

Reply:
@ArcturusZhang